hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/SecurityManagedConfiguration.java

```diff
@@ -477,6 +478,7 @@ protected void configure(final AuthenticationManagerBuilder auth) throws Excepti
     public static class RestSecurityConfigurationAdapter extends WebSecurityConfigurerAdapter {
 
         @Autowired
+        @Lazy
```

Comment:
Otherwise the startup of Hawkbit Update Server would fail with a BeanCurrentlyInCreationException. Using the Lazy annotation allows to resolve this circular dependency.